### PR TITLE
T-22 - Sending emails

### PR DESCRIPTION
**Pull request by sendr**
[T-22](https://youtrack.raccoongang.com/issue/T-22) - `sending emails`
- login validation - login user when both student and parent are active
- send Parent activation email with password
- removed extra empty option from the role select field

### CODE REVIEW

File: common/djangoapps/student/views/login.py [hunk 1]
```diff
@@ -465,6 +465,26 @@ def login_user(request):
                 # Important: This call must be made AFTER the user was successfully authenticated.
                 _enforce_password_policy_compliance(request, possibly_authenticated_user)
 
+        # Check whether both users (student and parent) are active
+        if possibly_authenticated_user and possibly_authenticated_user.is_active:
+            StudentProfile = apps.get_model('tedix_ro', 'StudentProfile')
+            ParentProfile = apps.get_model('tedix_ro', 'ParentProfile')
+            try:
+                # parent is logging in
+                if possibly_authenticated_user.parentprofile:
```
Comment (vladimir-seleznev) [inline common/djangoapps/student/views/login.py]: Why you use `if` here? You will get an exception for non-exists parent profile.

File: common/djangoapps/student/views/login.py [hunk 2]
```diff
@@ -465,6 +465,26 @@ def login_user(request):
...
+                    student = possibly_authenticated_user.parentprofile.students.filter(user__is_active=False).first()
+                    if student:
+                        _handle_failed_authentication(email_user, parent=True)
+            except ParentProfile.DoesNotExist:
+                # student is logging in
+                try:
+                    if possibly_authenticated_user.studentprofile:
```
Comment (vladimir-seleznev) [inline common/djangoapps/student/views/login.py]: Same as above

File: common/djangoapps/tedix_ro/forms.py [hunk 3]
```diff
@@ -11,8 +11,6 @@
 from tedix_ro.models import City, School, StudentProfile, CLASSROOM_CHOICES, InstructorProfile, ParentProfile
 
 ROLE_CHOICES = (
-    ('', ''),
```
Comment (vladimir-seleznev) [inline common/djangoapps/tedix_ro/forms.py]: Is it code appropriate to the current ticket?

Comment (sendr) [reply common/djangoapps/tedix_ro/forms.py]: no, it was a bug - at the Registration page under select Role we have two empty options, one from our custom form and another one from platform here - https://github.com/raccoongang/edx-platform/blob/tedix-hawthorn-stage/openedx/core/djangoapps/user_api/helpers.py#L250

File: common/djangoapps/student/views/login.py (hunk 1 above)
Comment (sendr) [reply common/djangoapps/student/views/login.py]: removed

File: common/djangoapps/student/views/login.py (hunk 2 above)
Comment (sendr) [reply common/djangoapps/student/views/login.py]: removed

**Pull request merged**